app/models/allocated_offender.rb

```diff
@@ -33,6 +33,21 @@ def new_case?
     }.any?
   end
 
+  def valid?
+    unless @offender.convicted?
+      Rails.logger.warn("[ALLOC] #{@offender.offender_no} has an allocation but is not convicted")
```

Comment:
There is no point logging warnings in our current infrastructure as they don't show up anywhere useful. We need to work out how we should raise such issues

Reply:
They turn up in Kibana if we log to STDOUT.

Reply:
but they will probably just get ignored - unless we have a process to look for warnings and errors in the Kibana logs

Reply:
That seems like too big a task for this PR.  If we want to know how many did or did not fail, then that's a job for metrics (PR incoming), but if we want to investigate detail, that's a logging thing.